*** x/cursesmodule/cursesmodule.c	Wed Mar 24 21:39:59 1999
--- cursesmodule.c	Thu Aug 19 01:22:27 1999
***************
*** 1395,1400 ****
--- 1395,1411 ----
    Py_INCREF(Py_None); \
    return Py_None; }
  
+ #define TwoArgNoReturnFunction(X, TYPE, PARSESTR) \
+ static PyObject * PyCurses_ ## X (self,arg) \
+      PyObject * self; \
+      PyObject * arg; \
+ { \
+   TYPE arg1, arg2; \
+   PyCursesInitialised \
+   if (!PyArg_Parse(arg,PARSESTR, &arg1, &arg2)) return NULL; \
+   Py_INCREF(Py_None); \
+   return PyCursesCheckERR(X(arg1, arg2), # X); }
+ 
  NoArgNoReturnFunction(beep)
  NoArgNoReturnFunction(def_prog_mode)
  NoArgNoReturnFunction(def_shell_mode)
***************
*** 1410,1415 ****
--- 1421,1428 ----
  NoArgNoReturnFunction(resetty)
  NoArgNoReturnFunction(savetty)
  
+ TwoArgNoReturnFunction(resizeterm, int, "(ii);y,x")
+ 
  NoArgOrFlagNoReturnFunction(cbreak)
  NoArgOrFlagNoReturnFunction(echo)
  NoArgOrFlagNoReturnFunction(nl)
***************
*** 2087,2092 ****
--- 2100,2106 ----
    {"raw",                 (PyCFunction)PyCurses_raw},
    {"reset_prog_mode",     (PyCFunction)PyCurses_reset_prog_mode},
    {"reset_shell_mode",    (PyCFunction)PyCurses_reset_shell_mode},
+   {"resizeterm",          (PyCFunction)PyCurses_resizeterm},
    {"setsyx",              (PyCFunction)PyCurses_setsyx},
    {"start_color",         (PyCFunction)PyCurses_Start_Color},
    {"termattrs",           (PyCFunction)PyCurses_termattrs},
